NodeJS/sample/SampleAgentClient.ts

```diff
@@ -55,6 +55,7 @@ async function deleteAgent(agentClient: AgentClient) {
 
 async function main() {
   const agentClient = await createAgentClient(endpoint, apiKey)
+  console.log('subaccount app id is ', agentClient.SubAccountAppId)
```

Comment:
I would strongly recommend NOT checking-in `console.log` statements, even if this is just a sample app. I won't block on it, but it is a code-smell to me.